.github/workflows/automerge.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: automerge
       uses: 'pascalgn/automerge-action@v0.13.1'
       env:
-        GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+        GITHUB_TOKEN: '${{ secrets.TOKEN }}'
```

Comment:
Could you please create to fix the token issue? And check if it works as expected?